src/components/app-demo/index.tsx

```diff
@@ -40,75 +31,33 @@ export class SmoothlyAppDemo {
+				</smoothly-app-room>
+				<smoothly-app-room path="button" label="Button">
 					<smoothly-button-demo />
-				</smoothly-room>
-				<smoothly-room path="dialog" label="Dialog">
+				</smoothly-app-room>
+				<smoothly-app-room path="dialog" label="Dialog">
 					<smoothly-dialog-demo />
-				</smoothly-room>
-				<smoothly-room path="display" label="Display" icon="eye-outline">
+				</smoothly-app-room>
+				<smoothly-app-room path="/display" label="Display" icon="eye-outline">
 					<smoothly-display-demo />
-				</smoothly-room>
-				<smoothly-room path="table" label="Table">
+				</smoothly-app-room>
+				<smoothly-app-room path="/table" label="Table">
 					<smoothly-table-demo />
-				</smoothly-room>
-				<smoothly-room path="select" label="Select">
+				</smoothly-app-room>
+				<smoothly-app-room path="/select" label="Select">
 					<smoothly-select-demo />
-				</smoothly-room>
-				<smoothly-room path="icon" label="Icon">
+				</smoothly-app-room>
+				<smoothly-app-room path="/icon" label="Icon">
 					<smoothly-icon-demo />
-				</smoothly-room>
-				<smoothly-room path="/redirect" label="Redirect">
-					<smoothly-button
-						style={{ "max-width": "300px" }}
-						onClick={() => {
-							redirect("/routing/pathParameter1")
-						}}>
-						Internal
-					</smoothly-button>
-					<smoothly-button
-						style={{ "max-width": "300px" }}
-						onClick={() => {
-							redirect("https://google.com")
-						}}>
-						External
-					</smoothly-button>
-				</smoothly-room>
-				<smoothly-room path={/^\/routing\/\w+\/?/} label="No effect">
-					<h2>Regex routing</h2>
-				</smoothly-room>
-				<smoothly-room path="old" label="Old" to="select"></smoothly-room>
-				<span slot="header" style={{ width: "100%", maxWidth: "500px" }}>
-					<smoothly-picker
-						label="All Animals Selected"
-						style={{ minWidth: "100px" }}
-						labelSetting="hide"
-						empty-menu-label="Sorry, we're out of options."
-						max-height="58px"
-						multiple={true}
-						select-none-name="Select All"
-						options={[
-							{ name: "Big Dog", value: "dog", aliases: ["WOFF"] },
-							{ name: "Cat Stevens", value: "cat", aliases: ["moew"], hint: "moew 🐈" },
-							{ name: "Noble Pig", value: "pig", hint: "🐷" },
-							{ name: "Turtle Wax", value: "turtle" },
-							{ name: "Spider Man", value: "spider" },
-							{ name: "Phoenix Order Long Wooord", value: "phoenix" },
-							{ name: "Horse Back", value: "horse" },
-							{ name: "Unicorn Horn", value: "unicorn" },
-							{ name: "Talking Parrot Parrot", value: "parrot" },
-							{ name: "Hidden Dragon", value: "dragon" },
-							{ name: "Scary Kraken", value: "kraken" },
-						]}></smoothly-picker>
-				</span>
-				<smoothly-trigger slot="header" type="link" name="logout">
+				</smoothly-app-room>
+				<smoothly-app-room path="/old" label="Old" to="select"></smoothly-app-room>
+				{/* <smoothly-trigger slot="header" type="link" name="logout">
```

Comment:
should this stay as a comment?

Reply:
We haven't really used the log out icon when fixing the nav bar but I'm not sure if it could be useful after